ql/src/java/org/apache/hadoop/hive/ql/optimizer/physical/Vectorizer.java

```diff
@@ -4962,9 +4969,8 @@ private static void createVectorPTFDesc(Operator<? extends OperatorDesc> ptfOp,
         evaluatorWindowFrameDefs,
         evaluatorInputExprNodeDescLists);
 
-    TypeInfo[] reducerBatchTypeInfos = vContext.getAllTypeInfos();
-
     vectorPTFDesc.setReducerBatchTypeInfos(reducerBatchTypeInfos);
+    vectorPTFDesc.setReducerBatchDataTypePhysicalVariations(reducerBatchDataTypePhysicalVariations);
```

Comment:
Shall we have setReducerBatchTypeInfos(Types, TypeVariations) instead of creating a separate method?
Seems like a good practice making sure we pas TypeVariations along with Types.

Reply:
Resolved thanks